examples/nitclk/multi-device-configured-acquisition-tclk.py

```diff
@@ -80,23 +80,27 @@
 sessions = []
 
 
-def check_for_error(client, vi, status):
-    """Raise an exception if the status indicates an error."""
-    if status != 0:
-        if client == niscope_client:
-            error_message_request = niscope_types.GetErrorMessageRequest(vi=vi, error_code=status)
-            error_message_response = niscope_client.GetErrorMessage(error_message_request)
-            raise Exception(error_message_response.error_message)
-        else:
-            error_message_request = nitclk_types.GetExtendedErrorInfoRequest()
-            error_message_response = nitclk_client.GetExtendedErrorInfo(error_message_request)
-            raise Exception(error_message_response.error_string)
+def check_for_scope_warning(response, vi):
+    """Print to console if the status indicates a warning."""
+    if response.status > 0:
+        warning_message = niscope_client.GetErrorMessage(
+            niscope_types.GetErrorMessageRequest(vi=vi, error_code=response.status)
+        )
+        sys.stderr.write(f"{warning_message.error_message}\nWarning status: {response.status}\n")
+
+
+def check_for_tclk_warning(response):
+    """Print to console if the status indicates a warning."""
+    if response.status > 0:
+        warning_message = nitclk_client.GetExtendedErrorInfo(
+            nitclk_types.GetExtendedErrorInfoRequest()
+        )
+        sys.stderr.write(f"{warning_message.error_string}\nWarning status: {response.status}\n")
 
 
 try:
-    # Initialize the session
-    i = 1
-    for resource in RESOURCES:
+    # Initialize the sessions
+    for i, resource in enumerate(RESOURCES, start=1):
```

Comment:
This is as good a place as any to comment: these examples look a lot better. Thanks!